src/styles/components/_about-us-introduction.scss

```diff
@@ -0,0 +1,95 @@
+.about-us-introduction {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  height: rem(250px);
+
+  @include respond-above('sm') {
+    height: rem(400px);
+  }
+  @include respond-above('md') {
+    height: rem(500px);
+  }
+
+  @include respond-above('lg') {
+    height: rem(600px);
+  }
+  @include respond-above('xl') {
+    height: rem(700px);
+  }
+
+  &__title {
+    width: 58%;
+    height: 12.5%;
+    margin-block-start: 0%;
+
+    color: $dark;
+    -webkit-text-stroke: rem(1px) $dark;
+
+    @include respond-above('md') {
+      font-size: rem(28px);
+    }
+
+    @include respond-above('lg') {
+      font-size: rem(32px);
+    }
+
+    @include respond-above('xl') {
+      width: 45%;
+
+      font-size: rem(36px);
+    }
+
+    font-size: rem(24px);
+    font-weight: bold;
+    font-stretch: normal;
+    font-style: normal;
+    line-height: 1.22;
+    letter-spacing: normal;
+    text-align: left;
+  }
+
+  &__description {
+    transform: translateY(-10%);
+    align-self: flex-end;
+    box-sizing: border-box;
+    width: 55%;
+    height: 55%;
```

Comment:
Hmm, somehow this height isn't enough for `width` around `1000px`. Try adjusting it along the way from mobile to desktop.

![Screen Shot 2564-02-26 at 01 27 39](https://user-images.githubusercontent.com/32289827/109200259-f8722600-77d2-11eb-9bfc-762c544b76b9.png)